preupg/application.py

```diff
@@ -489,13 +517,11 @@ def scan_system(self):
             log_message("The module {0} does not exist.".format(self.content))
             return ReturnValues.SCENARIO
         if not self.conf.contents:
-            version = SystemIdentification.get_assessment_version(
-                self.conf.scan)
-            if version is None:
-                log_message("Your scan is in a wrong format %s." % version,
-                            level=logging.ERROR)
-                log_message("It should be like 'RHEL6_7' for upgrade from"
-                            " RHEL 6->7.", level=logging.ERROR)
+            version = None
```

Comment:
Fixed in https://github.com/upgrades-migrations/preupgrade-assistant/pull/233/commits/107a550a8ec5aeb8ddcbd5e59fef0d6a99e3e0a2